app/views/layouts/application.html.erb

```diff
@@ -146,25 +146,16 @@
           </div>
           <div class="l-colspan--l colspan--l--has-border">
             <div class="footer__about">
-              <p>
-                <%= raw t('footer_about',
-                  publish_docs: "http://guides.rubygems.org/publishing/",
-                  install_docs: "http://guides.rubygems.org/command-reference/#gem-install",
-                  api_docs: "http://guides.rubygems.org/rubygems-org-api/",
-                  gem_list: rubygems_url,
-                  contributing_docs: "http://guides.rubygems.org/contributing/"
-                ) %>
-              </p>
-              <p>
-                RubyGems.org is made possible by
-                <%= link_to "many sponsors", page_path("sponsors") %>,
-                with development funded by the Ruby trade association,
-                <%= link_to "Ruby Together", "https://rubytogether.org/?source=rubygems" %>.
-              </p>
-              <p>
-                We need your help to fund the developer time that keeps RubyGems.org running smoothly for everyone.
-                <%= link_to "Join Ruby Together today", "https://rubytogether.org/join?source=rubygems" %>.
-              </p>
+              <%= raw t('footer_about',
+                publish_docs: "http://guides.rubygems.org/publishing/",
+                install_docs: "http://guides.rubygems.org/command-reference/#gem-install",
+                api_docs: "http://guides.rubygems.org/rubygems-org-api/",
+                gem_list: rubygems_url,
+                contributing_docs: "http://guides.rubygems.org/contributing/",
+                many_sponsors: page_path("sponsors"),
+                ruby_together: "https://rubytogether.org/?source=rubygems",
```

Comment:
Yes, I thought I'd add this to all language files. If it is not translated, it has the default language, English, as it was before